R/survDataCheck.R

```diff
@@ -151,14 +150,13 @@ survDataCheck <- function(data, diagnosis.plot = TRUE) {
   }
   # 7 test unique triplet Replicat - conc - time only if survDataCheck is called
   # by survData or reproData
-  # FIXME: replace ID creation by function call
-  ID <- str_c(data[, "replicate"], data[, "conc"], data[, "time"],
-              sep = "_")
-  if (any(duplicated(ID))) {
+  if (any(duplicated(idCreate(data, notime = FALSE)))) {
```

Comment:
The expression `idCreate(data, notime = FALSE)` is repeated 3 times. Please reintroduce an `ID` variable instead.